homeassistant/components/upnp/strings.json

```diff
@@ -15,9 +15,9 @@
       }
     },
     "abort": {
-      "already_configured": "UPnP/IGD is already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
       "no_devices_discovered": "No UPnP/IGDs discovered",
```

Comment:
Does this need a reference applied too?

Reply:
Yes we can ;)

Reply:
I think using `no_devices_found` in both cases makes sense.

Reply:
let's go for it then